packages/jsonforms-zorro-wrapper/src/controls/wysiwyg.renderer.ts

```diff
@@ -88,6 +111,26 @@ export class WysiwygRenderer extends JsonFormsControl {
       this.formats = this.uischema.options.formats || null;
     }
   }
+
+  override onChange(content: ContentChange) {
+    this.jsonFormsService.updateCore(Actions.update(this.propsPath, () => content.html));
+    if (this.uischema?.options.withStringValidation) {
+      this.jsonFormsService.updateCore(Actions.update(this.propsPath + this.stringFieldPostfix, () => content.text));
```

Comment:
I can see that `content.text` returns some enter at the end. 
![Screenshot 2024-08-07 at 15 33 25](https://github.com/user-attachments/assets/75bf1b2e-c035-459c-a7b2-392e93d40495)

What about trimming it?
```suggestion
      this.jsonFormsService.updateCore(Actions.update(this.propsPath + this.stringFieldPostfix, () => content.text.trim()));
```

Reply:
Trim will not solve this, because we got `\n` from ContentChange